src/main/scala/org.rogach.scallop/ScallopConfBase.scala

```diff
@@ -616,6 +616,46 @@ abstract class ScallopConfBase(
       .getOrElse(Right(()))
   }
 
+  def validateFilesExists(filesOption: ScallopOption[List[File]]) = addValidation {
+    filesOption.toOption
+	  .map(files => {
+        val problems = files.filterNot(_.exists)
+        if (problems.nonEmpty) Left(Util.format("File(s) %s not found", Util.seqstr(problems)))
+        else Right(())
+      })
+	  .getOrElse(Right(()))
+  }
+
+  def validateFilesDoesNotExist(filesOption: ScallopOption[List[File]]) = addValidation {
```

Comment:
English is not my native tongue - but I feel that `validateFilesDoNotExist` is more correct. (same thing 10 lines higher - `s` at the end is not necessary)